client/blocks/stats-navigation/index.js

```diff
@@ -29,6 +32,26 @@ class StatsNavigation extends Component {
 		isLegacy: PropTypes.bool,
 	};
 
+	state = {
+		isPopoverVisible: false,
```

Comment:
IMHO we could use a more meaningful name for Popover - for example `isPageSettingsPopoverVisible` or something?